Makefile

```diff
@@ -206,6 +206,10 @@ clean:
 	rm -rf dist __pycache__
 	rm -rf .coverage coverage_html_report
 
+	# Remove the standalone plugin 
+	rm -rf  $(MK_DIR)/mlir/build/lib/StandalonePlugin.*
+	rm -rf  $(MK_DIR)/mlir/standalone/build/lib/StandalonePlugin.*
+
```

Comment:
Maybe this should go into clean-mlir or clean-dialects (or be its own target and added to clean-all)? The clean target is not very well defined, but its mainly for the frontend/python package.

Reply:
_(or be its own target and added to clean-all)_
I think this is the best

Reply:
Done.